storage/indexer/src/db_indexer.rs

```diff
@@ -413,6 +482,22 @@ impl DBIndexer {
 
         assert_eq!(num_transactions, version - start_version);
 
+        if self.indexer_db.event_v2_translation_enabled() {
+            batch.put::<InternalIndexerMetadataSchema>(
```

Comment:
how do you plan to handle seq number backup and restore? since all data are in one db now?

Reply:
I will sync with you about this offline soon.

Reply:
The current plan is that we don't support the fast sync, but utilize the bootstrapping from the manual backup/restore.